examples/next/package.json

```diff
@@ -10,13 +10,14 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@aws-amplify/geo": "console-preview",
     "@aws-amplify/ui-react": "^5.3.1",
+    "@aws-amplify/ui-react-geo": "^1.0.2",
     "@aws-amplify/ui-react-liveness": "^2.0.8",
-    "@aws-amplify/ui-react-storage": "^2.3.1",
     "@aws-amplify/ui-react-notifications": "^1.0.13",
-    "@aws-amplify/ui-react-geo": "^1.0.2",
+    "@aws-amplify/ui-react-storage": "^2.3.1",
```

Comment:
weird, why was this so out of date

Reply:
Oh I think things just got reoredered so the versions look weird